.github/workflows/ci.yml

```diff
@@ -23,6 +24,6 @@ jobs:
         VALUESET_KEY: ${{ secrets.VALUESET_KEY }}
       run: unzip -P $VALUESET_KEY test/fixtures/EXM130-8.0.000/EXM130-8.0.000-bundle.zip
     - run: rm EXM130-8.0.000-bundle.json
-    - run: sudo chmod +x ./_tests_/setup.sh
-    - run: ./_tests_/setup.sh
-  
+    - run: sudo chmod +x ./src/_tests_/setup.sh
```

Comment:
Looks like you only chmod and never run the setup script.

We're still failing GH actions now it looks like the error is bc it can't find any test data (i.e. fpg doesn't exist)

Just run the setup script before running test_measure